it-index.html

```diff
@@ -0,0 +1,63 @@
+---
+title: "Jamulus ‒ Suonare Online. Tra Amici. In libertà."
+lang: "it"
+permalink: /
+layout: mainhomepage
+ldjAppCategory: "Musica"
+ldjSlogan: "Jamulus ‒ Suonare Online. Tra Amici. In libertà."
+ldjCreator: "Volker Fischer ed altri collaboratori"
+metadescription: "Jamulus è un software opensource che permette ai musicisti di suonare, provare o partecipare ad una jam, in tempo reale tramite Internet."
+mAltProgIcon: "Jamulus icon"
+mTSlogan: "Suonare Online. Tra Amici. In libertà."
+mTGetStartedNow: "Comincia a Suonare!"
+mHTMLDownloadNow: 'Advanced: <a href="https://sourceforge.net/projects/llcon/files/latest/download" target="_blank" id="dld_frm_sf" rel="noreferrer">Scarica Ora</a>'
+---
+<div class="fx-row fx-row-center-xs" id="firstrow">
+  <div class="fx-col-100-xs">
+    <div itemprop="abstract">
+      <h2>Cos'è Jamulus?</h2>
+      Jamulus è usato per suonare, provare, o partecipare ad una jam insieme ai tuoi amici, alla tua band o con chiunque trovi online.
+      Disponibile per Windows, macOS o Linux per connettersi ai server Jamulus di tutto il mondo, gratuitamente tramite una normale connessione internet a banda larga.
+      Oppure creare un server privato. Jamulus è progettato per un suono di alta qualità a bassa latenza, al fine di suonare con altri a tempo. Dai uno sguardo alla pagina  <a href="wiki/Demos" target="_blank"
+        rel="noreferrer"> che spiega come gli utenti usano Jamulus</a>!
+    </div>
+  </div>
+</div>
+<div class="fx-row fx-row-center-xs" id="bannercontainer">
+  <div class="fx-col-100-xs">
+    <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">
+      <img alt="Jamulus Banner. Links alla pagina Primi Passi" src="{{ '/assets/img/jamulusbannersmall.png' | relative_url }}" id="jamulusbanner" loading="lazy" />
+    </a>
+  </div>
+</div>
+<div class="fx-row fx-row-center-xs">
+  <div class="fx-col-100-xs fx-col-50-l">
+     <h2>Jamulus worldwide</h2>
```

Comment:
Same with "Jamulus worldwide"

Reply:
I updated Advanced word with right translation but Jamulus Worldwide is untranslatable.

Reply:
it remain the same in italian

Reply:
In the german version I also didn't use "Jamulus worldwide". I translated it into "Who uses Jamulus?" You don't need a word to word translation. Only the message which comes across needs to be similar.

Reply:
Yes i do not use world by world translation but in this case the meaning of worldwide is more big than who use jamulus, i think.
I think that rappresenting the jamulus community over the internet. One possible way to translate it is "Jamulus users over the world" ("Gli utenti Jamulus nel mondo").

You think is ok?

Reply:
Yeah. That could be a good option.

Reply:
ok i update!